test/unit/specs/components/Personalization/dom-actions/collectInteractions.spec.js

```diff
@@ -19,6 +19,7 @@ import cleanUpDomChanges from "../../../../helpers/cleanUpDomChanges.js";
 import { INTERACT_ID_DATA_ATTRIBUTE } from "../../../../../../src/components/Personalization/handlers/createDecorateProposition.js";
 import { getAttribute } from "../../../../../../src/components/Personalization/dom-actions/dom/index.js";
 import createDecoratePropositionForTest from "../../../../helpers/createDecoratePropositionForTest.js";
+import createRenderStatusHandler from "../../../../../../src/components/Personalization/handlers/createRenderStatusHandler.js";
```

Comment:
Generally, you don't want to import production code other than the code under test. You could mock the createRenderStatusHandler pretty easily with this:
```
export default {
      shouldRender: () => true,
      markAsRendered: () => {},
    };
```